.gitignore

```diff
@@ -19,3 +19,7 @@ torch_xla/csrc/aten_xla_type_default.cpp
 # Below files are not deleted by "setup.py clean".
 
 third_party/tensorflow/
+
+# Visual Studio Code files
+.vscode
```

Comment:
:)

Reply:
Why this change coming in within the TF subrepo?

Reply:
> Why this change coming in within the TF subrepo?

Never mind ... misread the diff 😄